doc/overview/dynamicCapabilities.md

```diff
@@ -118,4 +118,20 @@ and unload it when the condition is no longer satisfied.
             }
        }
  ```
-       
+
+Critical Capabilities Changes Error
+---------------------------------------------------
+With dynamic capabilities it is possible to get an error dialog during load that reads
+"Critical capabilities changes were detected...". This happens when an incompatible
+capability change occured during project load, and the project can never finish loading.
+The most typical cause of this is an unconfigured scope extension depends on a capability
+from the configured scope, but that extension point does not support dynamic capabilities.
```

Comment:
I think we should mention that capabilities from configured scope get merged into unconfigured, but that happens a bit later